lib/plugins/wallet/bitcoind/bitcoind.js

```diff
@@ -156,6 +180,7 @@ function newFunding (account, cryptoCode, settings, operatorId) {
       fundingConfirmedBalance,
       fundingAddress
     }))
+    .catch(console.log)
```

Comment:
```suggestion
    .catch(errorHandle)
```
Should we add this on all `newFunding`?